docker-compose.yml

```diff
@@ -51,7 +51,40 @@ services:
                 depends_on:
                         - "genesis_validator"
 
+        mongo:
```

Comment:
Wait, what's the reason for mongo here?

Reply:
It's the mongo service from the block explorer docker setup. Maybe a more suitable naming is needed?

Reply:
Oh, I didn't know block explorer was using mongo deep down. I just have personal bias against mongo : )